From 1ef9ab02af5c10a0fc7611c5579c4bc2344e83d0 Mon Sep 17 00:00:00 2001
From: Tomas Bzatek <tbzatek@redhat.com>
Date: Tue, 15 Feb 2022 13:25:22 +0100
Subject: [PATCH] doc: Fix @since: annotations

Recent changes in gdbus-codegen (2.71.1) have introduced parser
issues on malformed annotations.

https://gitlab.gnome.org/GNOME/glib/-/issues/2601
---
 data/org.freedesktop.UDisks2.xml                   | 2 +-
 modules/lvm2/data/org.freedesktop.UDisks2.lvm2.xml | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/data/org.freedesktop.UDisks2.xml b/data/org.freedesktop.UDisks2.xml
index 23f18746f..d9045a7f3 100644
--- a/data/org.freedesktop.UDisks2.xml
+++ b/data/org.freedesktop.UDisks2.xml
@@ -1802,7 +1802,7 @@
     <!-- Resize:
          @size: New partition size in bytes, 0 for maximal size.
          @options: Options.
-         @since 2.7.2
+         @since: 2.7.2
 
          Resizes the partition.
 
diff --git a/modules/lvm2/data/org.freedesktop.UDisks2.lvm2.xml b/modules/lvm2/data/org.freedesktop.UDisks2.lvm2.xml
index 46c8e5ac7..c1d61d44f 100644
--- a/modules/lvm2/data/org.freedesktop.UDisks2.lvm2.xml
+++ b/modules/lvm2/data/org.freedesktop.UDisks2.lvm2.xml
@@ -321,7 +321,6 @@
          @write_policy: Specifies the write policy.
          @options: Additional options.
          @result: The object path of the new VDO logical volume.
-
          @since: 2.9.0
 
          Create a new VDO logical volume that is backed by a newly created
